rotkehlchen/exchanges/data_structures.py

```diff
@@ -388,3 +389,33 @@ def trades_from_dictlist(
             continue
 
     return returned_trades
+
+
+binance_pair_db_tuple = Tuple[str, str, str, str]
```

Comment:
Capitalize this as per our typing convention
```suggestion
BINANCE_PAIR_DB_TUPLE = Tuple[str, str, str, str]
```